lib/features/campagne/campagne_state.dart

```diff
@@ -1,7 +1,9 @@
 import 'package:pass_emploi_app/models/campagne.dart';
+import 'package:pass_emploi_app/models/campagne_question_answer.dart';
 
 class CampagneState {
   final Campagne? campagne;
+  final List<CampagneQuestionAnswer> answers;
 
-  CampagneState(this.campagne);
+  CampagneState(this.campagne, this.answers);
```

Comment:
Pourquoi est-ce qu'on n'a pas fait différents states sur la campagne ? Genre present, absent, not initialized ?